plugins/source/aws/resources/services/config/config_rule_compliance_details.go

```diff
@@ -0,0 +1,65 @@
+package config
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/configservice"
+	"github.com/aws/aws-sdk-go-v2/service/configservice/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v2/schema"
+	"github.com/cloudquery/plugin-sdk/v2/transformers"
+)
+
+func configRuleComplianceDetails() *schema.Table {
+	tableName := "aws_config_config_rule_compliance_details"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/config/latest/APIReference/API_GetComplianceDetailsByConfigRule.html`,
```

Comment:
```suggestion
		Description: `https://docs.aws.amazon.com/config/latest/APIReference/API_EvaluationResult.html`,
```
Maybe? We generally link to the struct if there's a separate doc/type for it, and only link to the API call if the type is not separately documented.

Reply:
Yeah good point, probably better to change it.

Reply:
Other config resources may also have the same issue I don't think I paid attention to the description in those.